CHANGELOG.md

```diff
@@ -6,6 +6,83 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 
 ___Note:__ Yet to be released changes appear here._
 
+## 5.1.0
+
+### Main changes
+
+* Multiselect context pad added to our tools for quick deletion, alignement and spacing
+* Telemetry migrated to Mixpanel
```

Comment:
Not sure if this should be mentioned here. Our users don't benefit from the change.

Reply:
What we sometimes do is to add a one line summary in the beginning of the release (cf. https://github.com/bpmn-io/diagram-js/blob/develop/CHANGELOG.md#850). I'd encourage you to drop the "Main changes" section in favor of the existing pattern.